documentation/docs/index.md

```diff
@@ -1,7 +1,38 @@
 # Home
 
-Welcome to the ACCESS-OM3 wiki!
+Welcome to the documentation for the [ACCESS-OM3 global ice-ocean model configurations](https://github.com/ACCESS-NRI/access-om3-configs)!
 
-See the links on the right, particularly the [quick start](pages/Quick-start.md).
+See the links on the left, particularly the [quick start](pages/Quick-start.md).
+
+## Contributions welcome 🙏
+This documentation is a work in progress, and we welcome any contributions, including corrections and suggestions.
+
+Contributions can be made in a few ways.
+
+## Report a bug or make a suggestion
+Issues should be posted to [github.com/ACCESS-NRI/access-om3-configs/issues](https://github.com/ACCESS-NRI/access-om3-configs/issues). All contributions are welcome but we would particularly apprecaite text suggestions (below).
+
+## Quick contributions 
+!!! tip
+    For external people, this method does not allow one to preview the changes rendered into a website or create whole new pages. It's advantage is that it's *very quick* to do.<br>
+    For people with write and without write access to `ACCESS-OM3-CONFIGS`.<br>
+
+If a change is being made to an existing page, then the simplest and fastest way to make a change is to click the edit "pencil" on the top-right corner of each rendered page. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow one to edit the file. Once complete click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to `ACCESS-OM3-CONFIGS`: 
```

Comment:
```suggestion
The simplest and fastest way to make a change to an _existing_ page is to click the edit "pencil" on the top-right corner. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow you to edit the file. Once complete, click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs): 
```